src/R/Sql/Impl/Publish/DacPackageServices.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System.ComponentModel.Composition;
+using Microsoft.R.Components.Sql.Publish;
+
+namespace Microsoft.VisualStudio.R.Sql.Publish {
+    [Export(typeof(IDacPackageServices))]
```

Comment:
No, I'm not suggesting calling `new DacPackageServices()`, I thought about loading assembly and create an instance via `Activator`:
```{csharp}
var asm = Assembly.Load("Microsoft.VisualStudio.R.Sql");
var type = asm.GetTypes().FirstOrDefault(t => typeof(IDacPackageServices).IsAssignableFrom(typeof(C1)));
var instance = (IDacPackageServices)Activator.CreateInstance(type);
```